src/replayq.erl

```diff
@@ -573,8 +573,20 @@ get_commit_hist(Dir) ->
   CommitFile = commit_filename(Dir),
   case filelib:is_regular(CommitFile) of
     true ->
-      {ok, [#{segno := Segno, id := Id}]} = file:consult(CommitFile),
-      {Segno, Id};
+      case file:consult(CommitFile) of
+        {ok, [#{segno := Segno, id := Id}]} ->
+          {Segno, Id};
+        {ok, BadContent} ->
+          %% ignore empty file
+          %% sometimes (e.g. IOPS limited) do_commit (write to .tmp then rename) may fail
+          %% to write but succeed in rename.
+          %% ignoring COMMIT file leads to message receliver, so warning level
```

Comment:
do you think it worth to check if `COMMIT.tmp` exists and use it?

Reply:
don't think it worth the effort.